packages/react-scripts/config/webpack.config.js

```diff
@@ -538,6 +538,19 @@ module.exports = function(webpackEnv) {
       ],
     },
     plugins: [
+      // copy static assets from shared components
+      new CopyWebpackPlugin([
+        {
+          from: {
+            glob: path.join(
+              paths.appNodeModules,
+              `{@fs-sandbox,@fs}/**/dist/static/media/*`
+            ),
+          },
+          to: `static/media/[name].[ext]`,
```

Comment:
can we have naming conflicts here between modules that will stomp each other?

Reply:
Each name has a hash, so not likely.